.gitignore

```diff
@@ -25,7 +25,8 @@ docs/_build/
 *.user
 *.received.txt
 src/TestResults/
-src/TestDiffTools/Main.approved.txt
+src/TestDiffTools/*.approved.*
```

Comment:
We don't want to ignore these as it's good to have them visible in the PR to identity what's a breaking change.

Reply:
Do we still need Main.approved ignored? This is the only approval file in the utility for testing purposes.